### PR TITLE
Use GitHub API based sysl modules

### PR DESCRIPTION
**Pull request by ChloePlanet**
Works with
- `go run main.go --serve demo/demo.sysl --redoc --type=html`
- `make docker-mermaid-run`

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -12,20 +12,19 @@ test:
...
 docker:
 	docker build -t sysl-catalog .
 docker-mermaid:
 	docker build -t sysl-catalog-mermaid -f mermaid.Dockerfile . 
 docker-run: docker
-	docker run -it -v $$(pwd)/demo/markdown:/out:rw -v $$(pwd)/demo:/usr/demo:ro anzbank/sysl-catalog demo/sizzle.sysl
+	docker run -it -v $$(pwd)/demo/markdown:/out:rw -v $$(pwd)/demo:/usr/demo:ro anzbank/sysl-catalog demo/demo.sysl
 docker-mermaid-run: docker-mermaid
-	docker run -e SYSL_PLANTUML=localhost:8080 -v $$(pwd)/demo/html:/out:rw -v $$(pwd)/demo:/usr/demo:ro sysl-catalog-mermaid demo/sizzle.sysl --redoc --mermaid --type=html -o /out
+	docker run -e SYSL_PLANTUML=localhost:8080 -v $$(pwd)/demo/html:/out:rw -v $$(pwd)/demo:/usr/demo:ro sysl-catalog-mermaid demo/demo.sysl --redoc --mermaid --type=html -o /out
```
Comment (cuminandpaprika) [inline Makefile]: We're missing an environment variable here to use github mode for Sysl modules 👍 
```suggestion
	docker run -e SYSL_PLANTUML=localhost:8080 -e SYSL_MODULES=github -v $$(pwd)/demo/html:/out:rw -v $$(pwd)/demo:/usr/demo:ro sysl-catalog-mermaid demo/demo.sysl --redoc --mermaid --type=html -o /out
```

File: Makefile [hunk 2]
```diff
@@ -12,20 +12,19 @@ test:
...
+	sysl-catalog -o demo/markdown demo/demo.sysl --mermaid
 
 .PHONY: docker docker-mermaid
 docker:
 	docker build -t sysl-catalog .
 docker-mermaid:
 	docker build -t sysl-catalog-mermaid -f mermaid.Dockerfile . 
 docker-run: docker
-	docker run -it -v $$(pwd)/demo/markdown:/out:rw -v $$(pwd)/demo:/usr/demo:ro anzbank/sysl-catalog demo/sizzle.sysl
+	docker run -it -v $$(pwd)/demo/markdown:/out:rw -v $$(pwd)/demo:/usr/demo:ro anzbank/sysl-catalog demo/demo.sysl
```
Comment (cuminandpaprika) [inline Makefile]: And also here
```suggestion
	docker run -it -e SYSL_MODULES=github -v $$(pwd)/demo/markdown:/out:rw -v $$(pwd)/demo:/usr/demo:ro anzbank/sysl-catalog demo/demo.sysl
```

**Review by cuminandpaprika — COMMENTED** [review]
Looks good, still need to get to the bottom of this certificate issue

File: Makefile [hunk 3]
```diff
@@ -12,20 +12,18 @@ test:
...
 	docker build -t sysl-catalog .
 docker-mermaid:
 	docker build -t sysl-catalog-mermaid -f mermaid.Dockerfile . 
 docker-run: docker
-	docker run -it -v $$(pwd)/demo/markdown:/out:rw -v $$(pwd)/demo:/usr/demo:ro anzbank/sysl-catalog demo/sizzle.sysl
+	docker run -it -e SYSL_PLANTUML=localhost:8080 -e SYSL_MODULES=github -v $$(pwd)/demo/markdown:/out:rw -v $$(pwd)/demo:/usr/demo:ro anzbank/sysl-catalog demo/demo.sysl
 docker-mermaid-run: docker-mermaid
-	docker run -e SYSL_PLANTUML=localhost:8080 -v $$(pwd)/demo/html:/out:rw -v $$(pwd)/demo:/usr/demo:ro sysl-catalog-mermaid demo/sizzle.sysl --redoc --mermaid --type=html -o /out
-	cp demo/mastercard.yaml demo/html/demo/mastercard.yaml
+	docker run -e SYSL_PLANTUML=localhost:8080 -e SYSL_MODULES=github -v $$(pwd)/demo/html:/out:rw -v $$(pwd)/demo:/usr/demo:ro sysl-catalog-mermaid demo/demo.sysl --redoc --mermaid --type=html -o /out
```
Comment (cuminandpaprika) [inline Makefile:27]:
```suggestion
	docker run -e SYSL_PLANTUML=localhost:8080 -e SYSL_MODULES=github -v $$(pwd)/demo/html:/out:rw -v $$(pwd)/demo:/usr/demo:ro sysl-catalog-mermaid demo/demo.sysl --redoc --mermaid --type=html -o /out
	cp ~/.sysl/github.com demo/html/github.com
```

Comment (ChloePlanet) [reply Makefile:27]: Fixed by 987eb5d627e2bb0207793bbb7f8efdb77f66ef9d

**Pull request merged**